mhs/common/mhs_common/workflow/tests/test_sync_async.py

```diff
@@ -1,19 +1,26 @@
 from unittest import TestCase
-from unittest.mock import Mock, sentinel, patch
+from unittest.mock import Mock, sentinel, patch, MagicMock
 import utilities.message_utilities as message_utilities
 import mhs_common.messages.ebxml_envelope as ebxml_envelope
 import mhs_common.messages.ebxml_request_envelope as ebxml_request_envelope
 from mhs_common.workflow import sync_async
+from mhs_common.state import work_description as wd
+from utilities import test_utilities
 
 PARTY_ID = "PARTY-ID"
 MOCK_UUID = "5BB171D4-53B2-4986-90CF-428BE6D157F5"
 
 
+def throws_exception(x, y):
```

Comment:
Optional: unless you expect this function to be used elsewhere, I think it's better to be moved into the one test that it's used in. It feels a bit random it being here and accepting 2 random params.

Reply:
I've done as you suggested and also tried to name it a bit more descriptively